lib/core/test/integration/Test/Integration/Scenario/CLI/Wallets.hs

```diff
@@ -424,20 +463,107 @@ spec = do
             T.unpack err `shouldContain` "Passphrases don't match"
             exitCode `shouldBe` ExitFailure 1
 
-    it "WALLETS_UPDATE_PASSPHRASE_04 - \
-        \Cannot update passphrase if original passphrase is entered incorrectly"
-        $ \ctx -> do
+    describe "WALLETS_UPDATE_PASS_03 - Old passphrase values" $ do
+        let matrix =
+                [ ( show (passphraseMinLength - 1) ++ " char long"
+                  , replicate (passphraseMinLength - 1) 'ź'
+                  , expect (ExitFailure 1, mempty, "passphrase is too short")
+                  )
+                , ( show (passphraseMaxLength + 1) ++ " char long"
+                  , replicate (passphraseMaxLength + 1) 'ź'
+                  , expect (ExitFailure 1, mempty, "passphrase is too long")
+                  )
+                , ( "Empty passphrase"
+                  , ""
+                  , expect (ExitFailure 1, mempty, "passphrase is too short")
+                  )
+                , ( "Incorrect old passphrase", "wrong secure passphrase"
+                  , expect (ExitFailure 1, mempty, errMsg403WrongPass)
```

Comment:
I cringed when I saw the indentation on these two lines :grimacing:

Reply:
take it easy ;) [will fix]